features/FxA-16-devices-view/README.md

```diff
@@ -4,5 +4,19 @@ Device View - Basic UI
 
 https://mozilla.aha.io/features/FXA-16
 
-As A FF user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, record counts of different data types, and optionally data storage size (perhaps an option for self-hosters or others?)
+As a Firefox Accounts user when using Firefox, I want to know the devices and services I am connected to via my Firefox Account, and important information about my the status of my services. For devices, this includes last time synced, and optionally data storage size (perhaps an option for self-hosters or others?)
```

Comment:
nit: "about my the status"

I also think we could remove any mention of "services" from this description because it's really not obvious what that means.